app/controllers/api/motions_controller.rb

```diff
@@ -24,6 +24,12 @@ def update_outcome
     respond_with_resource
   end
 
+  def closed
+    load_and_authorize :group
+    @motions = @group.closed_motions.order(closed_at: :desc)
```

Comment:
Ah, I'd like to separate this out into `instantiate_collection`, so that we keep the pagination / timeframing capabilities that all the other endpoints have. I'll tap up a recommendation on how to do that in a bit.